src/Juvix/Data/Effect/Files/Pure.hs

```diff
@@ -84,10 +85,14 @@ re cwd = reinterpret $ \case
   RemoveFile' p -> removeFileHelper p
   RenameFile' p1 p2 -> renameFileHelper p1 p2
   CopyFile' p1 p2 -> copyFileHelper p1 p2
+  JuvixConfigDir -> return juvixConfigDirPure
   where
     cwd' :: FilePath
     cwd' = toFilePath cwd
 
+juvixConfigDirPure :: Path Abs Dir
```

Comment:
Should this be `Path Rel Dir` ? Because we expect it to be combined with an absolute root directory?

Reply:
This directory is only used in the pure implementation of `Files`. It has to be absolute, because that's what the `IO` implementation returns:
```
juvixConfigDirIO :: IO (Path Abs Dir)
juvixConfigDirIO = (<//> versionDir) . absDir <$> getUserConfigDir "juvix"
```

Reply:
got it - thank you!